ksqldb-functional-tests/src/test/resources/query-validation-tests/fk-join.json

```diff
@@ -155,6 +251,25 @@
         "message": "Invalid join condition: table-table joins require to join on the primary key of the right input table. Got LEFT_TABLE.ID1 = RIGHT_TABLE.F2."
       }
     },
+    {
+      "name": "Should fail on right non-key attribute for inner-join (reverse join condition order) - feature flag enabled",
+      "properties": {
+        "ksql.joins.foreign.key.enable": true
+      },
+      "comments": [
```

Comment:
If you go with my suggestion above to remove the copies of these tests without the feature flag enabled, then we can also remove these comments. When we ungate the feature flag, we'll simply remove the `properties` block rather than deleting the test. You can also update the test name to remove the `- feature flag enabled` portion now so we don't have to update the test name later when ungating the feature flag.